src/index.html

```diff
@@ -5,8 +5,387 @@
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="rate__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-title">Price:</p>
+              <p class="card__price-number">$2,199</p>
+            </div>
+            <a
+              class="card__link"
+              href="#"
+            >
+              <p class="card__link-text">BUY</p>
+            </a>
+          </div>
+
+          <div class="card">
+            <img
+              class="card__image"
+              src="./images/imac.jpeg"
+              alt="imac"
+            >
+
+            <p class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rate rate">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="rate__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-title">Price:</p>
+              <p class="card__price-number">$2,199</p>
+            </div>
+            <a
+              class="card__link"
+              href="#"
+            >
+              <p class="card__link-text">BUY</p>
+            </a>
+          </div>
+
+          <div class="card">
+            <img
+              class="card__image"
+              src="./images/imac.jpeg"
+              alt="imac"
+            >
+
+            <p class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rate rate">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <p class="rate__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price">
+              <p class="card__price-title">Price:</p>
+              <p class="card__price-number">$2,199</p>
+            </div>
+            <a
+              class="card__link"
+              href="#"
+            >
+              <p class="card__link-text">BUY</p>
+            </a>
+          </div>
+        </div>
+      </main>
+    </div>
   </body>
 </html>
```

Comment:
It's not a good idea to position these cards by wrapping 4 pieces of them. Cards must be easily added or removed in the future.

Reply:
I need every one of these tags)